### PR TITLE
Container Registry - code

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/container-registry.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-6/docs-ref-services/container-registry.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180024251160-107%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/container-registry.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-6/docs-ref-services/container-registry.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180050110898-107%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/container-registry.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-6/docs-ref-services/container-registry.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180406019392-107%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/container-registry.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-6/docs-ref-services/container-registry.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180418371643-107%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/container-registry.md [hunk 1]
```diff
@@ -13,42 +13,44 @@ ms.devlang: nodejs
...
+
+msRestAzure.interactiveLogin().then(credentials => {
+    let manager = new ContainerRegistryManagement(credentials, subscriptionId)
     return manager.registries.list()
-}).then( (registries) => {
+}).then(registries => {
     console.log("List of registries:")
     console.dir(registries, {depth:null, colors:true})
-}).catch( (error) => {
+}).catch(error => {
```
Comment (jawache) [inline docs-ref-services/container-registry.md]: Remove the error to be consistent with other node examples

File: docs-ref-services/container-registry.md [hunk 2]
```diff
@@ -13,42 +13,44 @@ ms.devlang: nodejs
...
 ```javascript
 const msRestAzure = require('ms-rest-azure')
 const ContainerRegistryManagement = require('azure-arm-containerregistry')
 
-msRestAzure.interactiveLogin().then( (credentials) => {
-    let manager = new ContainerRegistryManagement(credentials, 'your-subscription-id')
+const subscriptionId = 'your-subscription-id';
+
+msRestAzure.interactiveLogin().then(credentials => {
+    let manager = new ContainerRegistryManagement(credentials, subscriptionId)
```
Comment (jawache) [inline docs-ref-services/container-registry.md]: const manager

File: docs-ref-services/container-registry.md [hunk 3]
```diff
@@ -13,42 +13,44 @@ ms.devlang: nodejs
...
 
 ```javascript
 const msRestAzure = require('ms-rest-azure')
 const ContainerRegistryManagement = require('azure-arm-containerregistry')
 
-msRestAzure.interactiveLogin().then( (credentials) => {
-    let manager = new ContainerRegistryManagement(credentials, 'your-subscription-id')
+const subscriptionId = 'your-subscription-id';
+
+msRestAzure.interactiveLogin().then(credentials => {
```
Comment (jawache) [inline docs-ref-services/container-registry.md]: I think prettifier needs to be run on this code block to make it consistent...
```
 msRestAzure
   .interactiveLogin()
   .then(credentials => {....```

**Pull request merged**